src/Umbraco.Infrastructure/PublishedContentQuery.cs

```diff
@@ -237,10 +237,8 @@ private IEnumerable<IPublishedContent> ItemsByIds(IPublishedCache? cache, IEnume
         => ids.Select(eachId => ItemById(eachId, cache)).WhereNotNull();
 
     private IEnumerable<IPublishedContent> ItemsAtRoot(IPublishedCache? cache)
-    {
-        _documentNavigationQueryService.TryGetRootKeys(out IEnumerable<Guid> rootKeys);
-        return rootKeys.Select(x => cache?.GetById(false, x)).WhereNotNull();
-    }
+        => _documentNavigationQueryService.TryGetRootKeys(out IEnumerable<Guid> rootKeys) is false ? null!
```

Comment:
This cannot be null, it should just be an empty array 😁 

```suggestion
        => _documentNavigationQueryService.TryGetRootKeys(out IEnumerable<Guid> rootKeys) is false ? []
```